packages/fxa-settings/src/components/UnitRowSecondaryEmail/index.tsx

```diff
@@ -36,8 +37,18 @@ export const UnitRowSecondaryEmail = () => {
 
   const [alertBarRevealed, revealAlertBar, hideAlertBar] = useBooleanState();
 
+  // TODO: DRY this up and don't import `sentryMetrics` into every component
+  // needing a mutation - we must have an `onError` option in mutations
+  // to allow tests to pass but providing one prevents an error from actually
+  // throwing so we need to manually report it to Sentry.
+  // See https://github.com/apollographql/react-apollo/issues/2614
```

Comment:
Is there an issue filed for this follow up?

Reply:
I'll do this now, there's one other thing I need to file as well (noted in the "other information").

Reply:
Follow up for this is: FXA-2450
Follow up for what I've got in "other information" is: FXA-2449